freqtrade/commands/cli_options.py

```diff
@@ -647,4 +647,14 @@ def __init__(self, *args, **kwargs):
         nargs='+',
         default=[],
     ),
+    "freqaimodel": Arg(
+        '--freqaimodel',
+        help='Specify a custom freqaimodels.',
+        metavar='NAME',
+    ),
```

Comment:
maybe a stupid question - but especially the model is something that i assume is very tightly coupled with the strategy you're actually running. 
would it make sense to have that model be part of the strategy class (from a definition point of view)?

Reply:
Actually, they are very much decoupled. The user could run the same strategy on CatboostPredictionModel and LightGBMPredictionModel, or even a neuralnetwork. The models are agnostic to the strategy.

There is one strategy consideration that must be known (and it is already commented into the example strategy). If the user decides to create multiple targets, they would need to use the CatboostPredictionMultiModel (or LightGBMPredictionMultiModel) if they want to get multiple targets back. 

So it seems the current structure is best.